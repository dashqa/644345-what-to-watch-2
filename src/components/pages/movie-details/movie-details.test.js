import React from 'react';
import renderer from 'react-test-renderer';
import MovieDetails from './movie-details';

const props = {
  movie: {
    name: ``,
    genre: ``,
    released: 0,
    runTime: 0,
    backgroundImage: ``,
    posterImage: ``,
    director: ``,
    starring: [],
  },
  relatedMovies: []
};

it(`Movie details card correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieDetails {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
