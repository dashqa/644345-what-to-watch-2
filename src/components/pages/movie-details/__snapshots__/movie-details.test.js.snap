// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie details card correctly renders after relaunch 1`] = `
Array [
  <section
    className="movie-card movie-card--full"
    style={
      Object {
        "backgroundColor": undefined,
      }
    }
  >
    <div
      className="movie-card__hero"
    >
      <div
        className="movie-card__bg"
      >
        <img
          alt=""
          src=""
        />
      </div>
      <h1
        className="visually-hidden"
      >
        WTW
      </h1>
      <Header />
      <div
        className="movie-card__wrap"
      >
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              
            </span>
            <span
              className="movie-card__year"
            >
              0
            </span>
          </p>
          <MovieButtons />
        </div>
      </div>
    </div>
    <div
      className="movie-card__wrap movie-card__translate-top"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster movie-card__poster--big"
        >
          <img
            alt=""
            height="327"
            src=""
            width="218"
          />
        </div>
        <MovieCardTabs
          movie={Object {}}
        />
      </div>
    </div>
  </section>,
  <div
    className="page-content"
  >
    <Catalog
      movies={Array []}
    />
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          Â© 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
